src/cpp/fastdds/builtin/type_lookup_service/TypeLookupManager.cpp

```diff
@@ -695,6 +697,15 @@ const fastrtps::rtps::GUID_t& TypeLookupManager::get_builtin_request_writer_guid
     return c_Guid_Unknown;
 }
 
+void TypeLookupManager::advance_sequence_number() const
+{
+    ++request_seq_number_.low();
+    if (request_seq_number_.low() == 0)
+    {
+        ++request_seq_number_.high();
+    }
+}
```

Comment:
Why not use the RTPS sequence number which has this operation already defined and just assign the value to the DDS sequence number?